Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM python:3.8-alpine
-EXPOSE 8888
+FROM python:3.8-slim
+WORKDIR /mclocations
 
-WORKDIR /usr/src/app
+COPY requirements.txt .
+RUN pip install -r requirements.txt
 
-COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+COPY server.py .
+COPY mclocations mclocations/
 
-COPY server.py ./
+EXPOSE 8888
 
-CMD [ "python", "./server.py" ]
\ No newline at end of file
+ENTRYPOINT [ "python", "./server.py"]
```

Comment:
I was using entry point so I could run the docker command with `-vvv` on the end to get more logging.